src/components/shared/ContentSection/ContentSection.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const ContentSection = ({ cssClass, children }) => {
+  return <section className={cssClass}>{children}</section>;
```

Comment:
La clase que se usa es siempre `dp-container`, tiene sentido inyectarla?

Por otro lado, viste que hay formas de pasar todos los atributos sin ir uno por uno? Creo que es algo así:

```suggestion
const ContentSection = ({ children, ...rest }) => {
  return <section {...rest}>{children}</section>;
```

Reply:
en las paginas nuevas de planes no se usa siempre. @GusBaamonde

Reply:
y esto que te propuse del rest?

Reply:
si esto lo puedo agregar, me parecia mas claro sabiendo que se le pasa, pero bueno asi queda mas generico, se lo puedo cambiar